src/main/java/de/tum/in/www1/artemis/domain/ProgrammingExercise.java

```diff
@@ -712,6 +712,11 @@ public void validateStaticCodeAnalysisSettings(ProgrammingLanguageFeature progra
                     "staticCodeAnalysisNotSupportedForLanguage");
         }
 
+        // Check that FACT has no SCA enabled
+        if (Boolean.TRUE.equals(isStaticCodeAnalysisEnabled() && ProjectType.FACT.equals(getProjectType()))) {
```

Comment:
```suggestion
        if (Boolean.TRUE.equals(isStaticCodeAnalysisEnabled()) && ProjectType.FACT.equals(getProjectType())) {
```

Reply:
Wow.. this was actually hard to find.. Thanks 🙂